Description: Patch fixes call to Blt_ParseQualifiedName().
Author: FreeBSD BLT maintainers
Last-Modified: Tue, 08 Jul 2014 11:51:01 +0400

--- a/generic/bltNsUtil.c
+++ b/generic/bltNsUtil.c
@@ -64,7 +64,7 @@
     Tcl_Var varPtr;
     Tcl_Namespace *nsPtr;
     Tcl_Obj *objPtr;
-    char *str, *cp;
+    CONST char *str, *cp;
 
     varPtr = Tcl_FindNamespaceVar(interp, (char *)name, 
 	(Tcl_Namespace *)NULL, 0);
@@ -74,7 +74,7 @@
     objPtr = Tcl_NewObj();
     Tcl_GetVariableFullName(interp, varPtr, objPtr);
     str = Tcl_GetString(objPtr);
-    if (Blt_ParseQualifiedName(interp, str, &nsPtr, &cp) == NULL) {
+    if (Blt_ParseQualifiedName(interp, str, &nsPtr, &cp) != TCL_OK) {
         nsPtr = NULL;
     }
     Tcl_DecrRefCount(objPtr);
